ITEST/itest.storelogdata/pom.xml

```diff
@@ -17,6 +17,8 @@
     <properties>
         <surefire.argLine>-Xms128m -Xmx2048m -Xss256k</surefire.argLine>
         <maven.test.skip>true</maven.test.skip>
+        <testcontainers.version>1.17.5</testcontainers.version>
+
```

Comment:
Define 'version' properties and managing of dependencies in the root pom.xml file